### PR TITLE
[Graph Browser 2] Various Updates

### PR DESCRIPTION
**Pull request by chejennifer**
- added error handling
- updated the loading
- added node type to the title of browser pages
- updated the links on the landing page
- added headers for each section

the dev instance has been updated with these changes (eg. https://dev.datacommons.org/browser/geoId/12086)

### CODE REVIEW

**Review by beets — APPROVED** [review]
nice changes!

File: static/js/browser2/browser.ts [hunk 1]
```diff
@@ -32,16 +37,43 @@ window.onload = () => {
...
+      );
+    });
 };
+
+function getNodeType(
+  dcid: string,
+  listOfTypes: string[],
+  statVarId: string
+): string {
+  if (!_.isEmpty(statVarId)) {
```
Comment (shifucun) [inline static/js/browser2/browser.ts:72]: can just do "statVarID=="" here?

File: static/js/browser2/out_arc_section.tsx [hunk 2]
```diff
@@ -191,43 +210,53 @@ export class OutArcSection extends React.Component<
...
+      .then((resp) => {
+        const triplesData = resp.data;
+        const outArcs = triplesData.filter(
+          (t) => t.subjectId === this.props.dcid
+        );
+        const outArcsByPredProv: OutArcData = {};
+        for (const outArc of outArcs) {
+          const predicate = outArc.predicate;
+          if (IGNORED_OUT_ARC_PROPERTIES.has(predicate)) {
+            return;
```
Comment (shifucun) [inline static/js/browser2/out_arc_section.tsx:225]: need to remove spinner() before return?

File: static/js/browser2/weather_chart_section.tsx [hunk 3]
```diff
@@ -90,26 +96,35 @@ export class WeatherChartSection extends React.Component<
...
-        const values = {};
-        weatherData.forEach((data) => {
-          values[data.observationDate] = data.meanValue;
+    loadSpinner(LOADING_CONTAINER_ID);
+    Promise.all(weatherPromises)
+      .then((weatherPromisesData) => {
+        const propToSourceSeries = {};
+        weatherPromisesData.forEach((weatherData) => {
+          if (_.isEmpty(weatherData)) {
+            return;
```
Comment (shifucun) [inline static/js/browser2/weather_chart_section.tsx:105]: removeSpinner() before return?

File: static/js/browser2/arc_section.tsx [hunk 4]
```diff
@@ -107,7 +111,7 @@ export class ArcSection extends React.Component<
       })
       .catch(() => {
         this.setState({
-          errorMessage: `Error retrieving property labels for ${this.props.dcid}`,
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/browser2/arc_section.tsx:110]: done

File: static/js/browser2/browser.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/browser2/browser.ts]: done

File: static/js/browser2/out_arc_section.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/browser2/out_arc_section.tsx]: this return actually should be a continue, which I've updated

File: static/js/browser2/weather_chart_section.tsx (hunk 3 above)
Comment (chejennifer) [reply static/js/browser2/weather_chart_section.tsx:105]: return in a forEach is actually equivalent to continue

**Review by shifucun — APPROVED** [review]

**Pull request merged**